### DISCUSSION TITLE
Issue with conf editer via code option

### DISCUSSION THREAD

**Discussion by JohnFawkes** [Lissy93/dashy#242] [Help and Support] [answered]
Is it a bug or just on my end that if you go to edit the conf in dashy and switch to code veiw, the cursor is off by like 2 spaces. I cant actually go to the end of the line.

**Comment by lissy93** [comment, marked as answer]
Yeah, this is a bug- it's been raised in #124.
It happens in some browsers, and I think it's an issue with the JSON plugin I'm using, so I'll need to submt a PR to that project to fix it, although right now I'm not 100% sure what it's cause by. If you've got any ideas, please do let me know!

**Reply by lissy93** [reply]
✅ Fixed in #315
